sys/stl-compatibility/include/riot/chrono.hpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2015 Hamburg University of Applied Sciences (HAW)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup stl-compatibility
+ * @{
+ *
+ * @file    chrono.hpp
+ * @brief   STL chrono drop in replacement that adds the function now based on
+ *vtimer/timex
+ * @see     <a href="http://en.cppreference.com/w/cpp/thread/thread">
+ *            std::thread, defined in header <thread>
+ *          </a>
+ *
+ * @author  Raphael Hiesgen <raphael.hiesgen (at) haw-hamburg.de>
+ *
+ * @}
+ */
+
+#ifndef RIOT_CHRONO_HPP
+#define RIOT_CHRONO_HPP
+
+#include <chrono>
+#include <cstdio>
+#include <algorithm>
+
+#include "time.h"
+#include "vtimer.h"
+
+namespace riot {
+
+namespace {
+constexpr uint32_t microsecs_in_sec = 1000000;
```

Comment:
I added some comments for the `chrono.hpp` as it is not compliant. As for the other headers, I can add an additional link to the specific entry on cppreference. I believe there is not much to gain, as either my comments are so simple or colse to a copy of the reference.